sources/packages/backend/libs/integrations/src/institution-integration/ece-integration/ece-files/ece-file-detail.ts

```diff
@@ -0,0 +1,65 @@
+import { DisbursementValue } from "@sims/sims-db";
+import { END_OF_LINE, round, StringBuilder } from "@sims/utilities";
+import {
+  DATE_FORMAT,
+  ECERequestFileLine,
+  NUMBER_FILLER,
+  RecordTypeCodes,
+  SPACE_FILLER,
+} from "../models/ece-integration.model";
+
+/**
+ * Record of a ECE request file.
+ * The documentation about it is available on the document
+ * 'SIMSSFAS - Institution File layouts In Analysis Folder'.
+ */
+export class ECERequestFileDetail implements ECERequestFileLine {
+  transactionCode: RecordTypeCodes;
+  institutionCode: string;
+  disbursementValues: DisbursementValue[];
+  sin: string;
+  studentLastName: string;
+  studentGivenName: string;
+  birthDate: string;
+  sfasApplicationNumber: string;
+  institutionStudentNumber: string;
+  courseLoad: string;
+  studyStartDate: string;
+  studyEndDate: string;
+  disbursementDate: string;
+
+  getFixedFormat(): string {
+    const records = this.disbursementValues.map((disbursementValue) => {
+      const record = new StringBuilder();
+      record.append(this.transactionCode, 1);
+      record.append(this.institutionCode, 4);
+      record.appendWithStartFiller(
+        disbursementValue.id.toString(),
+        10,
+        NUMBER_FILLER,
+      );
+      record.append(disbursementValue.valueCode, 4);
+      record.appendWithStartFiller(
+        round(disbursementValue.valueAmount),
+        9,
+        NUMBER_FILLER,
+      );
+      record.append(this.sin, 9);
+      record.appendWithEndFiller(this.studentLastName, 25, SPACE_FILLER);
+      record.appendWithEndFiller(this.studentGivenName ?? "", 15, SPACE_FILLER);
+      record.appendDate(this.birthDate, DATE_FORMAT);
+      record.append(this.sfasApplicationNumber, 10);
```

Comment:
why is this called sfas application number? This integration is for sfas and sims both isn't it?